src/receta/controller/Chef.java

```diff
@@ -48,18 +49,29 @@ public double valorReceta(Receta[] recetas,String nombreReceta){
         return valor1;
     }
     
-    public void calcularValorReceta(Receta[] recetas, String ingrediente){
+    public void buscarRecetasPorIngrediente(Receta[] recetas, String ingrediente){
          for (int i = 0; i < recetas.length; i++){
-             for (int j = 0; j < recetas[i].getIngrediente().length; j++){
+             boolean encontroIngrediente = false;
+             for (int j = 0; j < recetas[i].getIngrediente().length && !encontroIngrediente; j++){ //Recorra mientras sea falso
                 if (recetas[i].getIngrediente()[j].getNombre().equals(ingrediente)){
-                    System.out.println("Puedes usar la receta: " + recetas[i].getNombre() + " con el ingrediente: " + ingrediente);
+                    encontroIngrediente = true;
                 }
              }
+             if (encontroIngrediente){
+                System.out.println("Puedes usar la receta: " + recetas[i].getNombre() + " con el ingrediente: " + ingrediente);
+             }else{
+                 System.out.println("Lo sentimos, no tenemos recetas con el ingrediente: " + ingrediente);

```

Comment:
Si es ninguna receta debe haber otro booleano e imprimir esto afuera de los dos for